plugins/node/opentelemetry-instrumentation-aws-lambda/README.md

```diff
@@ -86,6 +85,34 @@ The `lambdaHandler` should be specified as a string in the format `<file>.<handl
 
 One way to determine if the `lambdaHandler` option should be used is to check the handler defined on your Lambda. This can be done by determining the value of the `_HANDLER` environment variable or by viewing the **Runtime Settings** of your Lambda in AWS Console. If the handler is what you expect, then the instrumentation should work without the `lambdaHandler` option. If the handler points to something else, then the `lambdaHandler` option should be used to explicitly specify the handler that should be instrumented.
 
+### Active Tracing Context Propagation
+
+AWS Active Tracing can provide a parent context for the span generated by this instrumentation. In order to use this context, please use the [xray-lambda](https://github.com/open-telemetry/opentelemetry-js/tree/main/experimental/packages/propagator-aws-xray-lambda) propagator.
```

Comment:
> Can I report traces not to X-Ray and have AWS Active Tracing enabled? what should resolve this conflict?

If you report traces to a non-xray backend and use `xray-lambda` and enable active tracing, the reported traces will be broken as it will be missing the span generated by the lambda runtime that is only reported to xray. This can be fixed by disabling active tracing, or using just the `xray` propagator.